backend/es/es_index.py

```diff
@@ -44,8 +44,7 @@ def create(client, corpus_definition, add, clear, prod):
             sys.exit(1)
 
         logger.info('Adding prod settings to index')
-        index_settings = settings.get('index', {})
-        index_settings.update({
+        settings['index'].update({
             'number_of_replicas' : 0,
```

Comment:
Shouldn't we use `number_of_replicas: 1` in production? 🤔

Reply:
Only after the indexing procedure is complete. That happens at the end of the indexing script.